### ISSUE TITLE
check_pre_req.sh is in the wrong directory

### ISSUE THREAD

**Issue by MikeSpreitzer** [kubestellar/kubestellar#2697]
The `check_pre_req.sh` script is in `hack/` but belongs in `scripts/`. The former is for contributors, the latter is for all users.

**Comment by pratik-mahalle** [comment]
Yeap, the `check_pre_req.sh` is in the `hack/ `, Should I move it to the `scripts/`?

**Comment by MikeSpreitzer** [comment]
The script should be moved, but that is not something that we can do in one PR because we have some other scripts that reference the check_pre_req.sh script by a URL that points into a release on GitHub. I have created a PR, #2698, that takes the first steps in the move and puts in the release document instructions for what remains to be done later.

**Comment by pratik-mahalle** [comment]
I think, this seems big. We need to change the e2e script also as my tests are failing

**Comment by MikeSpreitzer** [comment]
Yes, we need to change the references. And we need to sequence the changes carefully across releases. See #2698, which seeks to take the first steps.
